src/index.js

```diff
@@ -1 +1,45 @@
 import './style.css';
+import NewPlayer from './newplaya';
+import moveToApi from './api';
+
+export default class Ui {
+  static render(players) {
+    const listItems = document.querySelector('.list');
+    players.forEach((player) => {
+      listItems.innerHTML += `
+      <li>
+        <p>${player.user}:</p>
+        <p> ${player.score}</p>
+      </li>
+      `;
+    });
+  }
+
+  static clearField() {
+    const listItems = document.querySelector('.list');
+    listItems.innerHTML = '';
+  }
+}
+
+const getData = async () => {
+  const pull = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Ml0wfxegyFf06umHiVlI/scores/');
+
+  const data = await pull.json();
+  const dataOut = await data.result;
+  Ui.render(dataOut);
+};
+
+const submitBtn = document.querySelector('.submitBtn');
+submitBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  const score = document.querySelector('.inputScore').value;
+  const name = document.querySelector('.inputName').value;
+  const player = new NewPlayer(name, Number(score));
+  moveToApi(player);
+});
```

Comment:
- [ ] [OPTIONAL] If you want to make your application look more professional it might be a nice idea to remove the form input after submitting, otherwise it will be very easy for people to spam and it will also make it more obvious that the score has been submitted, because, right now it looks like the application does nothing when clicking submit.